static/js/components/search/CountryRefinementOption.js

```diff
@@ -10,17 +10,22 @@ export default class CountryRefinementOption extends React.Component {
     count:    number,
   };
 
-  countryName = (countryCode: string): string => (
-    iso3166.country(countryCode).name
-  );
+  countryName = (countryCode: string): string => {
+    let country, countryName = "-";
+    if (countryCode) {
+      country = iso3166.country(countryCode);
+      countryName = country ? country.name : "-";
+    }
+    return countryName;
```

Comment:
^^ I think the best thing would be to fix the function in `lib/currency` and then use it here (also, add a test for the `codeToCountryName` function which tests the case when the code is `null` or `undefined`).